### ISSUE TITLE
AttributeError: 'module' object has no attribute 'rnn_cell'

### ISSUE THREAD

**Issue by true-source** [tensorflow/magenta#538]
I was trying out one of the pretrained models (basic_rnn) using the melody_rnn_generate.py scirpt and is encountering the above issue. I looked up the corresponding script in the site-packages directory and found that it is using 'tf.nn.rnn_cell.BasicLSTMCell' which i believe was written for a previous version. I tried updating the python2 magenta package by 'pip2 install magenta --upgrade' but it didn't seem to solve the issue. Seems like python package for magenta hasn't been updated for tensorflow 1.0.

**Comment by cghawthorne** [comment]
Yes, you are correct. Unfortunately we need to figure out a solution to #529 before we cut a new release. I'm hoping that will be very soon.

**Comment by zhxsxuan** [comment]
Any temporary solution for now? I tried to down grade tensorflow to 0.12.1. But it still doesn't work.

**Comment by brannondorsey** [comment]
In case anyone else runs into this before its fixed, I was able to get around this issue w/ the most recent magenta installed via pip by uninstalling tensorflow 1.0 and instead installing tensorflow 0.12. :

```
pip install magenta
pip uninstall tensorflow
#pip uininstall tensorflow-gpu # uncomment if using gpu
pip install tensorflow==0.12.0
#pip install tensorflow-gpu==0.12.0 # uncomment if using gpu
```

**Comment by dovanchan** [comment]
Traceback (most recent call last):
  File "/Users/DOVAN/anaconda2/envs/magenta/bin/image_stylization_transform", line 11, in <module>
    sys.exit(console_entry_point())
  File "/Users/DOVAN/anaconda2/envs/magenta/lib/python2.7/site-packages/magenta/models/image_stylization/image_stylization_transform.py", line 144, in console_entry_point
    tf.app.run(main)
  File "/Users/DOVAN/anaconda2/envs/magenta/lib/python2.7/site-packages/tensorflow/python/platform/app.py", line 43, in run
    sys.exit(main(sys.argv[:1] + flags_passthrough))
  File "/Users/DOVAN/anaconda2/envs/magenta/lib/python2.7/site-packages/magenta/models/image_stylization/image_stylization_transform.py", line 127, in main
    os.path.expanduser(FLAGS.input_image)), 0)
  File "/Users/DOVAN/anaconda2/envs/magenta/lib/python2.7/posixpath.py", line 254, in expanduser
    if not path.startswith('~'):
AttributeError: 'NoneType' object has no attribute 'startswith'


I have try tensorflow0.12.0 and tensorflow 1.0,it still no work~

**Comment by brannondorsey** [comment]
That looks to me like a different error, where the `path` module in `posixpath.py` is somehow `None`.

**Comment by dovanchan** [comment]
I have search many information about this,but It still no work~ Can you tell me what this error means particularly? 'NoneType' object has no attribute 'startswith'.  Sincerely~

**Comment by dovanchan** [comment]
Traceback (most recent call last):
  File "/Users/DOVAN/anaconda2/envs/magenta/bin/image_stylization_transform", line 11, in <module>
    sys.exit(console_entry_point())
  File "/Users/DOVAN/anaconda2/envs/magenta/lib/python2.7/site-packages/magenta/models/image_stylization/image_stylization_transform.py", line 144, in console_entry_point
    tf.app.run(main)
  File "/Users/DOVAN/anaconda2/envs/magenta/lib/python2.7/site-packages/tensorflow/python/platform/app.py", line 44, in run
    _sys.exit(main(_sys.argv[:1] + flags_passthrough))
  File "/Users/DOVAN/anaconda2/envs/magenta/lib/python2.7/site-packages/magenta/models/image_stylization/image_stylization_transform.py", line 135, in main
    _multiple_images(image, which_styles, output_dir)
  File "/Users/DOVAN/anaconda2/envs/magenta/lib/python2.7/site-packages/magenta/models/image_stylization/image_stylization_transform.py", line 88, in _multiple_images
    tf.concat_v2([input_image for _ in range(len(which_styles))], 0),
AttributeError: 'module' object has no attribute 'concat_v2'

When I try to do this:
(magenta) DOVANtekiMacBook-Pro:/ DOVAN$ sudo image_stylization_transform       --num_styles=10       --checkpoint=/Users/DOVAN/Downloads/multistyle-pastiche-generator-monet.ckpt       --input_image=/Users/DOVAN/Desktop/photo.jpg       —which_styles="{0:0.1,1:0.1,2:0.1,3:0.1,4:0.1,5:0.1,6:0.1,7:0.1,8:0.1,9:0.1}
"       --output_dir=/Users/DOVAN/Desktop/       --output_basename="stylized"


It appear a new error :AttributeError: 'module' object has no attribute 'concat_v2'
 How to slove it~

**Comment by dovanchan** [comment]
I foud that problem right now!!Because if you install the magenta right now,the version of tensorflow is 1.00(default), But the code is using tensorflow0.12 ,So we must `pip uninstall tensorflow`and `pip install tensorflow==0.12.0`;That 's all

If you still wanna use tensorflow1.0,try this 

> ;In the short-term (before 1.0 final happens) concat_v2 will be the function to use. Once 1.0 final happens, concat_v2() will be renamed to concat() and then what used to be concat() will cease to exist.

 https://github.com/tensorflow/tensorflow/issues/6345

**Comment by cghawthorne** [comment]
We've released version 0.1.10 of our pip package, which should solve these problems. We've also updated our development environment to fix the bazel import problems we were having (as long as you have Bazel 0.4.5). Can you try installing the latest version and see if you still have issues?
